scripts/actions/fetch-and-deserialize.js

```diff
@@ -40,15 +40,20 @@ const fetchAndDeserialize = (accessToken) => async ({ locale, fileUris }) => {
   const zipEntries = zip.getEntries();
 
   const translatedHtml = zipEntries.map((entry) => {
+    console.log(entry.entryName);
```

Comment:
Was this meant to be logged? If so, should this include our little `[*]` denotation?

Reply:
No it was just for testing purposes! It's removed now :)